lib/rubycritic/command_factory.rb

```diff
@@ -4,9 +4,18 @@ module Rubycritic
   class CommandFactory
     def self.create(options = {})
       Config.set(options)
+      validate_rails_directory if Config.for_rails
```

Comment:
Yeah, that is the thing that we need to watch for: All of a sudden our generic CommandFactory contains Rails specific code which decreases cohesion and increases coupling. How about creating commands/rails.rb?